packages/lab/src/stepper-input/useStepperInput.ts

```diff
@@ -131,24 +131,24 @@ export const useStepperInput = (
       setCurrentValue(roundedValue);
     }
 
-    if (onChange) {
-      onChange(roundedValue);
+    if (onChange && event) {
+      onChange(event, roundedValue);
```

Comment:
Does this mean button long press will not trigger the event? 

is this captured in test?